tools/test-proxy/scripts/transition-scripts/README.md

```diff
@@ -8,16 +8,46 @@ The script `generate-assets-json.ps1` will execute the initial migration of your
 
 The script is [generate-assets-json.ps1](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/transition-scripts/generate-assets-json.ps1)
 
+### Download the transition script locally
+
+```powershell
+Invoke-WebRequest -OutFile "generate-assets-json.ps1" https://raw.githubusercontent.com/Azure/azure-sdk-tools/main/tools/test-proxy/scripts/transition-scripts/generate-assets-json.ps1
+```
+
+```bash
+wget https://raw.githubusercontent.com/Azure/azure-sdk-tools/main/tools/test-proxy/scripts/transition-scripts/generate-assets-json.ps1
+```
+
 ## Setup
 
-Before running the script, understand that **only services that have migrated to use the `test-proxy` as their record/playback solution can store recordings into the external assets repository.** The CLI command commands are invoked from the `test-proxy` itself, so if it is NOT being used for record/playback, the work to transition to the `test-proxy` must be done before recordings can be moved.
+Before running the script, understand that **only services that have migrated to use the `test-proxy` as their record/playback solution can store recordings into the external assets repository.** The test-proxy itself contains the code for `restoring`/`push`ing recordings, so if it is NOT being used for record/playback, that work must be completed before recordings can be moved.
```

Comment:
I think the code blocks are confusing here, because `restoring` is not a test proxy verb. Maybe just remove them, or an alternative?

```suggestion
Before running the script, understand that **only services that have migrated to use the `test-proxy` as their record/playback solution can store recordings into the external assets repository.** The test-proxy itself contains the code for restoring/pushing recordings (`test-proxy [restore|push]`). If it is NOT being used for record/playback, that work must be completed before recordings can be moved.
```